azerothcore/azerothcore-wotlk
# fix(Core/DB): Hardcoded Loch Modan texts

Removed the Loch modan script from the core and Rammed it into the database (Pun intended).





## CHANGES PROPOSED:
-  Removed Mountaineer Pebblebitty's script from the core
- Added gossip interaction to the database
- Added missing gossip when player has the key
- Added smart script to close the dialog


## ISSUES ADDRESSED:
- Closes part of #2444



## TESTS PERFORMED:
- Compiled and tested




## HOW TO TEST THE CHANGES:

.go c 8369
talk to npc and see if gossip menu options work (close in the last one)
.additem 10000 (Accept quest)
End quest on mountaineer and accept the next one
.quest reward 3182
.quest add 3201
End quest on mountaineer (Gives the key to searing gorge)
Talk to her again and check for a new gossip.

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
